### PR TITLE
fix: style clean up & test tooling versions

### PR DESCRIPTION
**Pull request by Westbrook**
## Description
- fix oversight in RTL implementation of Sidenav Item
- modify Menu Item demo in preparation for future browser issues.
- update test tooling to latest which allow us to remove some `webkit` failure forking due to the latest version of Playwright. This may require `rm -rf ~/Library/Caches/ms-playwright && yarn install` to ensure you have the latest version of browsers locally for testing `number-field` going forward.

## Motivation and Context
A pending update to Playwright futzes with the ligatures in Menu Item demos.

## How Has This Been Tested?
Visual regressions

## Screenshots (if appropriate):
https://westbrook-sidenav-rtl-light-medium-rtl--spectrum-web-components.netlify.app/review/

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:
- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: packages/menu/stories/menu-item.stories.ts [hunk 1]
```diff
@@ -41,18 +41,24 @@ export const noWrap = (): TemplateResult => {
...
             </sp-menu-item>
             <sp-menu-item selected>
                 Save As...
-                <kbd slot="value">⇧⌘S</kbd>
+                <kbd slot="value">⇧​⌘​S</kbd>
             </sp-menu-item>
             <sp-menu-item disabled>
                 Save All
-                <kbd slot="value">⌥⌘S</kbd>
+                <kbd slot="value">⌥​⌘​S</kbd>
```
Comment (adixon-adobe) [inline packages/menu/stories/menu-item.stories.ts:61]: Curious if you know why this changed (text editor differences?)

**Review by adixon-adobe — APPROVED** [review]
One question I'm only curious about, but changes look good!

File: packages/menu/stories/menu-item.stories.ts (hunk 1 above)
Comment (Westbrook) [reply packages/menu/stories/menu-item.stories.ts:61]: I wish I could explain...

It's even hard to see what the actual change here is. Between the characters in each `<kbd>` element I've added a `zero width space` character. The theory is that it prevent ligatures. as I was getting different sizes of the `⌘` character in the first two menu items than in the third one, which was really weird:

image

I made the related CSS changes after the fact, so I'm running a couple branches to make sure. One without the zero width spaces, but the CSS changes, to see if that's enough. Namely thinking that it'll be hard to know you removed the zero width space if you edit this file. And, another with no zero width spaces and `font-variant-ligatures: none;` added to be more specific about it. Then I'll know for sure, hopefully, and can make a note in the story file.

Comment (adixon-adobe) [reply packages/menu/stories/menu-item.stories.ts:70]: If you do need to keep the zero-width spaces, I'd definitely add a comment to indicate that's expected/there.

Comment (adixon-adobe) [reply packages/menu/stories/menu-item.stories.ts:70]: But I hope they're not 😮

**Pull request merged**